salt/metalk8s/addons/addons-service-config/deployed.sls

```diff
@@ -0,0 +1,5 @@
+# Create the namespace before deploying configmap objects
```

Comment:
This is exactly why the default configs belong with the addons and not here.

Reply:
ok